nix/modules/devShells.nix

```diff
@@ -172,26 +170,34 @@
             '';
           };
 
-        rustDev =
-          mkRustShell
-            {
-              inputsFrom = [
-                self'.packages.holochain
-              ];
+        rustDev = pkgs.mkShell {
+          inputsFrom = [
+            self'.packages.holochain
+          ];
 
-              shellHook = ''
-                export CARGO_HOME="$PWD/.cargo"
-                export CARGO_INSTALL_ROOT="$PWD/.cargo"
-                export CARGO_TARGET_DIR="$PWD/target"
-                export CARGO_CACHE_RUSTC_INFO=1
-                export PATH="$CARGO_INSTALL_ROOT/bin:$PATH"
-                export NIX_PATH="nixpkgs=${pkgs.path}"
-                export PS1='\n\[\033[1;34m\][rustDev:\w]\$\[\033[0m\] '
-                echo Rust development shell spawned. Type 'exit' to leave.
-              '' + (lib.strings.optionalString pkgs.stdenv.isDarwin ''
-                export DYLD_FALLBACK_LIBRARY_PATH="$(rustc --print sysroot)/lib"
-              '');
-            };
+          packages = (lib.lists.optionals pkgs.stdenv.isLinux [
+            pkgs.mold
+            pkgs.pkgsStatic.openssl
```

Comment:
i'm not clear on why this is necessary to add explicitly. it should be introduced by having the holochain package in `inputsFrom`. 

while i'm not clear on why exactly it works i can confirm it does and so i accept it as a mysterious yet working solution.

Reply:
This is more serious for people who are building on Windows. It's multiple error-prone steps to get OpenSSL provided and found for linking on Windows. If something has broken with our feature flags to ensure OpenSSL gets built as part of the Rust build then we need to follow up on that. Could you open a GitHub issue or similar please? I'm happy for it to be assigned to me as long as we are tracking it